src/components/loader/loader-square.test.tsx

```diff
@@ -22,7 +22,7 @@ afterAll(() => {
 });
 
 it.each([0, 1, 2])("each loader square renders as expected", (index) => {
-  mockUseMediaQuery.mockReturnValueOnce(true);
+  mockUseMediaQuery.mockReturnValueOnce(false);
```

Comment:
comment: the should be `true`, but as there is a `beforeEach` at the top of the file already you don't need to have any mock value here